### PR TITLE
RF_04.001.02 | Folder > Rename Folder | Rename folder from drop-down menu

### PR DESCRIPTION
**Pull request by yanaoreshkevich**
Implemented Changes:

Created new file **folderRenameFolderPOM.cy.js**
Converted **TC_04.001.02 to POM** format.

To **DashboardPage.js**:

- added locator getRenameFolderDropdownMenuItem
- added method clickRenameFolderDropdownMenuItem()

To **FolderPage.js**:

- added locators getNewNameField, getFolderUrl
- added methods clearNewNameField(), typeNewFolderName(), verifyFolderUrl()

**[RF_04.001.02](https://github.com/RedRoverSchool/JenkinsQA_JS_2024_fall/issues/506)**

wjfEHWlQrI

### CODE REVIEW

**Review by ivanlashkevich — APPROVED** [review]
Nice work done! Approved.

File: cypress/e2e/FolderRenameFolderPOM.cy.js
Comment (Negrescu29) [inline cypress/e2e/FolderRenameFolderPOM.cy.js:1]: shouldn't the name of the file `cypress/e2e/FolderRenameFolderPOM.cy.js` start with a lowercase letter

like: folderRenameFolderPOM.cy.js

File: cypress/e2e/FolderRenameFolderPOM.cy.js [hunk 1]
```diff
@@ -0,0 +1,38 @@
...
+import NewJobPage from "../pageObjects/NewJobPage";
+import FolderPage from "../pageObjects/FolderPage";
+import genData from "../fixtures/genData";
+import Header from "../pageObjects/Header";
+
+const dashboardPage = new DashboardPage();
+const newJobPage = new NewJobPage();
+const folderPage = new FolderPage();
+const header = new Header();
+const folderName = genData.newProject();
```
Comment (Negrescu29) [inline cypress/e2e/FolderRenameFolderPOM.cy.js:13]: nice use of genData ❤️

File: cypress/e2e/FolderRenameFolderPOM.cy.js [hunk 2]
```diff
@@ -0,0 +1,38 @@
+/// <reference types="cypress"/>
+
+import DashboardPage from "../pageObjects/DashboardPage";
+import NewJobPage from "../pageObjects/NewJobPage";
+import FolderPage from "../pageObjects/FolderPage";
+import genData from "../fixtures/genData";
+import Header from "../pageObjects/Header";
```
Comment (Negrescu29) [inline cypress/e2e/FolderRenameFolderPOM.cy.js:7]: small nit, but maybe change the order so Pages are all together and fixtures are divided with a empty line

**Review by Negrescu29 — CHANGES_REQUESTED** [review]
lgtm. need to change the `cypress/e2e/FolderRenameFolderPOM.cy.js` file name to start with a lowercase

File: cypress/e2e/FolderRenameFolderPOM.cy.js
Comment (yanaoreshkevich) [reply cypress/e2e/FolderRenameFolderPOM.cy.js:1]: Yes, you're rigth! Yes, you are right! Thanks for your observation. I'll fix it now

File: cypress/e2e/FolderRenameFolderPOM.cy.js (hunk 2 above)
Comment (yanaoreshkevich) [reply cypress/e2e/FolderRenameFolderPOM.cy.js]: Done!

**Pull request merged**